tron/trondaemon.py

```diff
@@ -180,10 +179,18 @@ def run(self):
             self._run_www_api()
             self._run_manhole()
             self._run_reactor()
+            self._run_eventbus()
+
+    def _run_eventbus(self):
+        self.eventbus = make_eventbus(f"{self.options.working_dir}/_events")
+        # self.eventbus.start()
```

Comment:
?

Reply:
yeah, don't want to actually run it when this PR is merged. i'll add relevant args / configuration options separately.